src/trusted-types.js

```diff
@@ -0,0 +1,26 @@
+const policyOptions = {
+    createHTML: html => html,
+    createScript: script => script,
+};
+const { policy, policyEnabled } = createPolicy();
+
+function createPolicy() {
+    if (typeof trustedTypes !== "undefined") {
+        try {
+          return { policy: trustedTypes.createPolicy("esmoduleshims", policyOptions), policyEnabled: true};
+        } catch {
+
+        }
+    }
+  return { policy: policyOptions, policyEnabled: false };
+}
```

Comment:
Since we do try to save bytes here might be worth flattening this:

```suggestion
export let policy;
if (typeof trustedTypes !== "undefined") {
  try {
    policy = trustedTypes.createPolicy("esmoduleshims", policyOptions);
  } catch {}
}
```

And then replacing `policyEnabled` with just `if (policy)` kind of thing.

Also I think we need to change the implementations of `trustedInnerHTML` and `trustedScript` to work when there is no policy too, see comments below.